src/browser/modules/Frame/FrameTitlebar.tsx

```diff
@@ -163,6 +161,17 @@ function FrameTitlebar(props: FrameTitleBarProps) {
   }, [props.frame.isRerun])
   useEffect(gainFocusCallback, [gainFocusCallback])
 
+  useEffect(() => {
```

Comment:
I'm curious, is this related to removing the modal somehow?

Reply:
No it's related to be being sloppy :/ good you caught it